airbyte-integrations/connectors/source-facebook-marketing/integration_tests/spec.json

```
@@ -5,14 +5,17 @@
     "title": "Source Facebook Marketing",
     "type": "object",
     "properties": {
-      "account_id": {
-        "title": "Ad Account ID",
-        "description": "The Facebook Ad account ID to use when pulling data from the Facebook Marketing API. The Ad account ID number is in the account dropdown menu or in your browser's address bar of your <a href=\"https://adsmanager.facebook.com/adsmanager/\">Meta Ads Manager</a>. See the <a href=\"https://www.facebook.com/business/help/1492627900875762\">docs</a> for more information.",
+      "account_ids": {
+        "title": "Ad Account ID(s)",
+        "description": "The Ad account ID number is in the account dropdown menu or in your browser's address bar of your <a href=\"https://adsmanager.facebook.com/adsmanager/\">Meta Ads Manager</a>. See the <a href=\"https://www.facebook.com/business/help/1492627900875762\">docs</a> for more information.",
         "order": 0,
         "pattern": "^[0-9]+$",
-        "pattern_descriptor": "1234567890",
+        "pattern_descriptor": "The Ad Account ID must be a number.",
         "examples": ["111111111111111"],
-        "type": "string"
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
```

```suggestion
        "items": {
          "type": "string",
          "pattern": "^[0-9]+$",
        }
```
Pattern should be moved inside items

---

Fixed